amethyst_gltf/src/format/material.rs

```diff
@@ -125,11 +125,12 @@ fn deconstruct_image(data: &TextureData, offset: usize, step: usize) -> TextureD
         TextureData::Image(ref image_data, ref metadata) => {
             let metadata = metadata
                 .clone()
-                .with_size(image_data.raw.w as u16, image_data.raw.h as u16)
+                .with_size(image_data.rgba.width() as u16, image_data.rgba.height() as u16)
                 .with_format(SurfaceType::R8);
             let image_data = image_data
-                .raw
-                .buf
+                .rgba
+                .clone() // Should this be avoided? The function takes data by reference.
```

Comment:
It's fine, it's just during load and not during use.

Reply:
Removing the comment.